src/screens/ConfirmSignUp.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { Auth } from 'aws-amplify';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import AuthTextInput from '../components/AuthTextInput';
+import AuthButton from '../components/buttons/AuthButton';
+import { useNavigation } from '@react-navigation/native';
+
+const ConfirmSignUp: React.FC = () => {
+    const [username, setUsername] = useState('');
+    const [authCode, setAuthCode] = useState('');
+    const navigation = useNavigation();
+    async function confirmSignUp() {
```

Comment:
this should have loading functionality so we can disable the button when the asynchronous work is happening, check out the follower button for a good example of how to do this